### ISSUE TITLE
Additional properties

### ISSUE THREAD

**Issue by oscargus** [apytypes/apytypes#18]
One should probably think about the names but:

* Largest positive/negative value
* Unit of least position (fixed)
* Smallest positive/negative value (float)

**Comment by Theodor-Lindberg** [comment]
One should maybe have two version of these in the case for floating-point, one for normal numbers and one for subnormal numbers?

E.g. something like this
```Python
x = APyFloat(0, 0, 0, exp_bits=4, man_bits=3)
x.max_pos_normal # 448
x.max_pos_subnormal # 0.875 * 2^-6
x.min_neg_normal # -2^-6
x.min_neg_subnormal # -2^-9
```

**Comment by oscargus** [comment]
I have slightly changed my mind about these... Especially for floats it is not so obvious. However, for fixed-point a max and min property would make sense, as well as ulp. 

But with that said, your suggestion makes sense. It would probably also be interesting to know the smallest normalized value as well. The sign one can probably ignore in the float case? (As it is pretty obvious...) Maybe ulp should be for that particular number and not the representation?

**Comment by oscargus** [comment]
Hard to make it consistent for both float and fixed...

**Comment by Theodor-Lindberg** [comment]
Yes the sign can probably ignored, otherwise the API might feel cluttered rather than consistent. Also, having the ulp for a particular number and not the representation makes sense like you suggest.

Maybe some of these should be available as static functions as well in the floating-point case? That way one wouldn't need to create an object just to find out some properties about a format.
```Python
APyFloat.max_normal(exp_bits, man_bits)
APyFloat.max_subnormal(exp_bits, man_bits)
```
